### PR TITLE
Added the launch folder for Octopus 2022 Q3

### PR DESCRIPTION
**Pull request by ellapradella**
# About

This is a post about...

# Release

Is this post for a specific release? If so which one?

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here.

# Pre-requisites

- [ ] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [ ] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

### CODE REVIEW

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+This release introduces a new GitHub Action: [Push Build Information](https://github.com/marketplace/actions/push-build-information-to-octopus-deploy). This GitHub Action lets you provide detailed commit, build, and issue tracking information to Octopus, which can be surfaced as release notes. 
+
+You can use this feature to share release notes with stakeholders for each production deployment. You can also follow the links from deployments back to builds and commits when trying to isolate bugs that were introduced in specific releases. 
+
+Learn more about [Build Information and Issue Tracking](https://octopus.com/docs/packaging-applications/build-servers/build-information).
+
+## Easier onboarding for teams
+
+[Watch our video that demonstrates easier onboarding for teams](https://youtu.be/EHByk8vG8xM).
```
Comment (TeganAli) [inline blog/2022-09/octopus-release-2022-q3/index.md:215]:
```suggestion
[Watch our video showing you how easy it is to get started with Octopus Deploy](https://youtu.be/EHByk8vG8xM).
```

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+This release introduces a new GitHub Action: [Push Build Information](https://github.com/marketplace/actions/push-build-information-to-octopus-deploy). This GitHub Action lets you provide detailed commit, build, and issue tracking information to Octopus, which can be surfaced as release notes. 
+
+You can use this feature to share release notes with stakeholders for each production deployment. You can also follow the links from deployments back to builds and commits when trying to isolate bugs that were introduced in specific releases. 
+
+Learn more about [Build Information and Issue Tracking](https://octopus.com/docs/packaging-applications/build-servers/build-information).
+
+## Easier onboarding for teams
+
+[Watch our video showing you how easy it is to get started with Octopus Deploy](https://youtu.be/EHByk8vG8xM).
```
Comment (TeganAli) [inline blog/2022-09/octopus-release-2022-q3/index.md:215]:
```suggestion
[Watch our video walk-through that helps you deploy with Octopus in minutes](https://youtu.be/EHByk8vG8xM).
```

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+Learn more about our [ServiceNow integration](https://octopus.com/docs/approvals).
+
+> The ServiceNow integration is available as part of our Enterprise subscription to help larger organizations with their change management. Contact our sales team for a demo and [learn more about Octopus Deploy pricing](https://octopus.com/pricing).
+
+## Efficient change management approvals with Jira Service Management (early access preview)
+
+![Screenshot of Jira Service Management integration](jira-task-settings.png "width=500")
+
+Build on our ITSM change management support, our Jira Service Management integration is available as an early access preview (EAP). This integration links Octopus deployments to Jira Service Management change requests, making it easier to manage deployment pipelines at scale. It reduces the need for manually filling out change requests, making the process quicker and reducing errors.
```
Comment (robpearson) [inline blog/2022-09/octopus-release-2022-q3/index.md:121]:
```suggestion
Building on our ITSM change management support, our Jira Service Management integration is available as an early access preview (EAP). This integration links Octopus deployments to Jira Service Management change requests, making it easier to manage deployment pipelines at scale. It reduces the need for manually filling out change requests, making the process quicker and reducing errors.
```

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+- Aggregate data across your space so you can compare metrics across projects to identify what's working and what isn’t
+- Inform better decision making: identify problems, track improvements, and celebrate successes 
+- Help you quantify DevOps performance based on what's actually happening as shown by data
+
+Learn more about [how Space level insights can help your team make better decisions](https://octopus.com/docs/insights). 
+
+> Space level insights are available as a part of the Octopus Deploy Enterprise subscription. Contact our sales team for a demo and to [learn more about Octopus Deploy pricing](https://octopus.com/pricing).
+
+### Customer feedback from the DevOps Insights Alpha Program
+
```
Comment (robpearson) [inline blog/2022-09/octopus-release-2022-q3/index.md:173]: I'd probably add a line here saying something about dozens of customers participated in our DevOps insights alpha program and we thank them for their time and feedback. This has shaped the feature for thousands of other customers.

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+This release introduces a new GitHub Action: [Push Build Information](https://github.com/marketplace/actions/push-build-information-to-octopus-deploy). This GitHub Action lets you provide detailed commit, build, and issue tracking information to Octopus, which can be surfaced as release notes. 
+
+You can use this feature to share release notes with stakeholders for each production deployment. You can also follow the links from deployments back to builds and commits when trying to isolate bugs that were introduced in specific releases. 
+
+Learn more about [Build Information and Issue Tracking](https://octopus.com/docs/packaging-applications/build-servers/build-information).
+
+## Easier onboarding for teams
+
+[Watch our video walk-through that helps you deploy with Octopus in minutes](https://youtu.be/EHByk8vG8xM).
```
Comment (robpearson) [inline blog/2022-09/octopus-release-2022-q3/index.md:215]: I'd replace the link with an embedded video. 

<iframe width="560" height="315" src="https://www.youtube.com/embed/EHByk8vG8xM" frameborder="0" allowfullscreen></iframe>

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+
+![Simpler cloud target discovery setup in Octopus Deploy](cloud-target-discovery-azure.png "width=500")
+
+We made it easier to configure cloud connections so you can deploy to Azure, AWS, and Google Cloud. You can now configure cloud connections directly in the deployment process editor. This means you can configure and select the appropriate cloud account settings in fewer clicks. You'll also be informed when it's configured.
+
+Learn more about how to [configure cloud connections](https://octopus.com/docs/infrastructure/deployment-targets/cloud-target-discovery).
+
+## Breaking changes 
+
+This release contains two breaking changes:
```
Comment (robpearson) [inline blog/2022-09/octopus-release-2022-q3/index.md:235]: This is now 3. :)

**Review by robpearson — APPROVED** [review]

File: blog/2022-09/octopus-release-2022-q3/index.md (hunk 4 above)
Comment (ellapradella) [reply blog/2022-09/octopus-release-2022-q3/index.md:182]: Added thanks

File: blog/2022-09/octopus-release-2022-q3/index.md (hunk 5 above)
Comment (ellapradella) [reply blog/2022-09/octopus-release-2022-q3/index.md]: This feature is no longer part of Q3

File: blog/2022-09/octopus-release-2022-q3/index.md (hunk 6 above)
Comment (ellapradella) [reply blog/2022-09/octopus-release-2022-q3/index.md]: Thanks

File: blog/2022-09/octopus-release-2022-q3/index.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+- Inform better decision making: identify problems, track improvements, and celebrate successes 
+- Help you quantify DevOps performance based on what's actually happening as shown by data
+
+Learn more about [how Space level insights can help your team make better decisions](https://octopus.com/docs/insights). 
+
+> Space level insights are available as a part of the Octopus Deploy Enterprise subscription. Contact our sales team for a demo and to [learn more about Octopus Deploy pricing](https://octopus.com/pricing).
+
+### Customer feedback from the DevOps Insights Alpha Program
+
+We had several customers participate in our DevOps Insights Alpha Program and we thank you for your time and feedback. This has helped to shaped the feature for thousands of other customers.
```
Comment (TeganAli) [inline blog/2022-09/octopus-release-2022-q3/index.md:183]:
```suggestion
Several customers participated in our DevOps Insights Alpha Program, and we thank them for their time and feedback. They helped shape the feature for thousands of other customers.
```

**Pull request merged**